zer0-os/zOS
# feat: expose sender avatar in channel view

### What does this do?
- exposes sender avatar in channel view.
- test coverage.

### Why are we making this change?
- product request.

### How do I test this?
- open a channel and check sender avatar is displayed (following the group convention)

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


https://github.com/zer0-os/zOS/assets/39112648/4279265f-14d4-45bc-8b6b-6f1c47fe15a7
